.github/workflows/lint-and-test.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         include:
           - python-version: "3.9"
             coverage: yes
+            upgrade: yes
```

Comment:
I'd call this `upgrade-dependencies`

Reply:
Resolving this as I'm moving the CI change into another PR.